File: problemas/1100/1113.cs
using System;

namespace uri1113
{
    internal static class Program
    {
        private static void Main()
        {
            while (true)
            {
                string[] entrada = Console.ReadLine().Split(' ');
                int x = int.Parse(entrada[0]);
                int y = int.Parse(entrada[1]);

                if (x == y)
                {
                    break;
                }

                Console.WriteLine(x < y ? "Crescente" : "Decrescente");
            }
        }
    }
}

File: problemas/1100/1113.dart
import 'dart:io';

void main() {
  while (true) {
    final entrada = stdin.readLineSync().split(' ');
    final x = int.parse(entrada[0]);
    final y = int.parse(entrada[1]);

    if (x == y) {
      break;
    }

    stdout.writeln(x < y ? 'Crescente' : 'Decrescente');
  }
}